bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/i18n/broadlink_xx.properties

```diff
@@ -0,0 +1,21 @@
+# FIXME: please substitute the xx with a proper locale, ie. de
+# FIXME: please do not add the file to the repo if you add or change no content
```

Comment:
You need to follow this text, either delete the file if you change no content, or you need to rename it to _en.properties for english. Once you have done it, remove the fixme

Reply:
Re-generated the internationalization file.